ray_autoscale.yaml

```diff
@@ -0,0 +1,101 @@
+# cluster.yaml =========================================
+
+# An unique identifier for the head node and workers of this cluster.
+cluster_name: eugene_test #<YOUR NAME>
```

Comment:
would you mind changing this to not have my name

Reply:
Done.